docs/README.md

```diff
@@ -3,3 +3,20 @@
 ## Browser support
 
 We align our browser support targets with that of Wagtail. Have a look at the [official documentation](http://docs.wagtail.io/en/latest/contributing/developing.html).
+
+## Python/Django/Wagtail support
+
+Supporting Python versions as defined in `setup.py` classifiers.
+
+Supporting Django versions as defined in the Django project [supported versions](https://www.djangoproject.com/download/#supported-versions).
+
+Supporting Wagtail versions from `1.7`.
```

Comment:
I couldn't find a proper list of Wagtail's supported versions. The [release process](http://docs.wagtail.io/en/latest/contributing/release_process.html) document seems to imply that only the current version and the previous one will receive bug and security fixes but it also mentions LTS and I couldn't find a definite list of such versions. Therefore leaving the arbitrary `1.7` we started with.